### DISCUSSION TITLE
ConnectionRadius in ReactFlow is not working in version 11

### DISCUSSION THREAD

**Discussion by sudarsanarao995** [xyflow/xyflow#3492] [General]
bellow image want to work as Expecting Result 
![Capture](https://github.com/wbkd/react-flow/assets/147405464/4aba5bdd-f4dd-4f2a-98c7-2ab96c7c51af)

**Comment by moklick** [comment]
Please provide a codesandbox. The connectionRadius should, as seen here: https://codesandbox.io/s/react-flow-starter-typescript-j8lkh

**Reply by sudarsanarao995** [reply]
![image](https://github.com/wbkd/react-flow/assets/147405464/df7cb39d-a605-4168-8beb-581024d9aae6)
Added the connectionRadius = {20} , but not reflecting any connectionRadius changes 

you can see codesandbox added here connectionRadius property : https://codesandbox.io/s/wandering-mountain-fcs89p?file=/src/Flow.tsx

**Reply by moklick** [reply]
20 is the default value. When you set it to 100 for example, you can see that the connection happens even if you are 100px away from a handle https://codesandbox.io/s/nifty-zeh-fxqv7f?file=/src/Flow.tsx:1303-1332

**Reply by sudarsanarao995** [reply]
yes, when it is set to 100 connection happening but user wants below image that circular radius , when it is comes closer the connection will  happen automatic 
![image](https://github.com/wbkd/react-flow/assets/147405464/fc52dce9-95c6-4a27-b95f-03ecfe387771)

**Reply by moklick** [reply]
Sorry, I don't get the issue here. What are you expecting to happen?
